lib/cli/node-modules-path.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var path  = require('path');
+
+module.exports = function(context) {
+  var nodePath = process.env.EMBER_NODE_PATH;
+  var contextPath = path.resolve(context);
+
+  if (nodePath) {
+    var nodeModulesPath = path.resolve(nodePath);
+
+    if (contextPath.indexOf(nodeModulesPath) === 0) {
+      return path.resolve(contextPath,'node_modules');
```

Comment:
This seems to use `contextPath` when it is more specific than `nodeModulesPath`.

For example given a `contextPath` of `/Projects/cat-time` and a `nodeModulesPath` of `/Projects`, the `/Projects/cat-time` directory would be used (`/Projects` has an index of `0` on `/Projects/cat-time`).

This seems backwards? And perhaps we should wrap this in a small semantic helper called `hasSubdirectoryOf` etc.